apps/web-portal-client/lib/features/examples/components/TippyExample.jsx

```diff
@@ -0,0 +1,37 @@
+import React, {useState} from 'react'
+import Tippy from '@tippy.js/react'
+import 'tippy.js/dist/tippy.css'
```

Comment:
I think in our case it makes sense to choose the Tippy library, it's much easier to use

However, we need to decide on where we would include `css` dependencies, I mean the one place to do that.